ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -146,7 +146,7 @@ tests:
   steps:
     cluster_profile: gcp-qe
     env:
-      ENABLE_OTA_TEST: OCP-24358
+      ENABLE_OTA_TEST: OCP-24358,OCP-56083
```

Comment:
Per our original design, `ENABLE_OTA_TEST` should only support "false", "true", and single case id, but no multiple case id. cc @jiajliu

Reply:
Yes, correct. if OCP-56083 is not for a case which needed to be ran on a dedicate cluster with other test excluded, we could add it into run_ota_multi_test while setting ENABLE_OTA_TEST: true, which means all the cases in the caseset would be ran on the cluster.

Reply:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-commands.sh#L284

Reply:
Thanks @jiajliu @jianlinliu, updated in latest commit